package.json

```diff
@@ -84,6 +84,7 @@
     "gulp-clean-css": "^2.0.11",
     "gulp-concat": "^2.6.0",
     "gulp-concat-css": "^2.3.0",
+    "gulp-debug": "^2.1.2",
```

Comment:
Lets not introduce additional dependencies. Maybe we can report caching using standard `console.log` / `gulp.utils` or just not report it at all?